longevity_test.py

```diff
@@ -352,7 +352,10 @@ def chunks(_list, chunk_size):
             # add few stress threads with tables that weren't pre-created
             customer_profiles = self.params.get('cs_user_profiles', default=[])
             for cs_profile in customer_profiles:
-                for i in range(4):
+                num_of_newly_created_table = 4
```

Comment:
I thought we said let's add 1 every time.
I'll merge it, but if the problem persist let's reduce to one.

Reply:
I thought we talked about doing pre-create each round.
but lets first run it, and we'll see.
we have the new 3.2 RC ready ?